project/Dependencies.scala

```diff
@@ -59,7 +59,8 @@ object Dependencies {
     "org.jsoup" % "jsoup" % "1.12.1" % scope,
     "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
     "org.mockito" %% "mockito-scala-scalatest" % "1.7.1" % scope,
-    "org.scalacheck" %% "scalacheck" % scalacheckVersion % scope
+    "org.scalacheck" %% "scalacheck" % scalacheckVersion % scope,
+    "com.typesafe.akka" %% "akka-testkit" % "2.5.18" % scope
```

Comment:
Use `akkaVersion` variable. We already overrides some dependency versions.

Reply:
Done in https://github.com/hmrc/voa-bar-xml-frontend/pull/182/commits/df4ae3ff0ee0f0cd586b554468c66def6ca01d55